src/styles/index.css

```diff
@@ -0,0 +1,68 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider using SCSS variables for the font family and other repeated values to enhance maintainability and consistency across your styles.